### PR TITLE
Update to Get-IntuneConfigurationProfileSettings.ps1

### PR DESCRIPTION
**Pull request by danr2929**
Updated to page through graph to return all settings found in settings catalog configuration profiles. Originally only returned first 25 settings.

### CODE REVIEW

File: Source/ConfigurationProfile/Get-IntuneConfigurationProfileSettings.ps1 [hunk 1]
```diff
@@ -43,19 +43,25 @@ function Get-IntuneConfigurationProfileSettings
...
     }
     process {
-        
 
-        $response = Invoke-MgRestMethod -Method Get -Uri "$uri/$graphVersion/deviceManagement/configurationPolicies('$Id')/settings" -OutputType Json | ConvertFrom-Json
+        $configurations = @()
+        $reqeustUri     = "$uri/$graphVersion/deviceManagement/configurationPolicies('$Id')/settings"
 
+        do{
+            $response = Invoke-MgRestMethod -Method Get -Uri $reqeustUri -OutputType JSON | convertFrom-json
```
Comment (kenjohnson03) [inline Source/ConfigurationProfile/Get-IntuneConfigurationProfileSettings.ps1]: Let's add '-Depth 50' here to ensure that we don't miss anything with future values that may come back.

**Review by kenjohnson03 — COMMENTED** [review]
Added comment. Please update and resubmit.

**Review by danr2929 — COMMENTED** [review]
updated line 51 to include ConvertTo-JSON so -depth of 50 can be specified. Has to be done this way to support lower PS versions.

**Comment by kenjohnson03** [comment]
Looks great!

**Pull request merged**